docs/css/README.md

```diff
@@ -79,10 +181,10 @@ We organize our partials into separates folders:
 | settings   | Variables & common settings                                                                                           | Usually one `_settings` file                                      |
```

Comment:
That underscore syntax is heavily borrowed from preprocessors. Could we maybe be more agnostic in that table ? or make a difference between preprocessors and CSS/PostCSS